scripts/ci-subtask.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# ./ci-subtask.sh <TOTAL_TASK_N> <TASK_INDEX>
+
+packages=(`go list ./...`)
+dirs=(`find . -iname "*_test.go" -exec dirname {} \; | sort -u | sed -e "s/^\./github.com\/tikv\/pd/"`)
+tasks=($(comm -12 <(printf "%s\n" "${packages[@]}") <(printf "%s\n" "${dirs[@]}")))
+
+weight () {
+    [[ $1 == "github.com/tikv/pd/server/api" ]] && return 30
+    [[ $1 == "github.com/tikv/pd/server/tso" ]] && return 30
```

Comment:
All TSO tests are in the `tests` folder, and they are running independently through `tso-function-test`. I think we should exclude them here.

```suggestion
    [[ $1 == "github.com/tikv/pd/tests/server/tso" ]] && return 30
```